azurerm/resource_arm_action_group_test.go

```diff
@@ -0,0 +1,242 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMActionGroup_basic(t *testing.T) {
```

Comment:
can we re-suffix this test to `_complete` and also add tests for each of the receivers independently? e.g. `_emailReceivers`, `_smsReceivers`, `_webhookReceivers`

in addition: can we add an update test here, e.g. `basic` -> `complete` -> `basic` to ensure these can be added/removed?